src/js/jsx/sections/layers/LayersPanel.jsx

```diff
@@ -284,9 +149,12 @@ define(function (require, exports, module) {
                     <div
                         ref="container"
                         className={containerClasses}
-                        onClick={this._handleContainerClick}
-                        onClickCapture={this._handleContainerClickCapture}>
-                        {childComponents}
+                        onClick={this._handleContainerClick}>
+                        <LayersGroup
+                            isRoot={true}
+                            disabled={this.props.disabled}
+                            document={doc}
+                            layerNodes={doc.layers.roots}/>
```

Comment:
I suggest just naming this prop `layers`, just because it's simpler and easier to understand. Also, the parameter is actually a list of `Layer` models and not `LayerNode` models.